CHANGELOG.rst

```diff
@@ -1,6 +1,51 @@
 Changelog
 =========
 
+0.5.0 (2018-08-17)
+------------------
+
+MLflow 0.5.0 includes some major improvements, including Keras and PyTorch first-class support as models, SFTP support as an artifactory, an improved scatterplot visualization to compare runs, and a more complete Python SDK for low-level experiment and run management.
+
+Breaking changes:
+
+- The Tracking API has been split into two pieces, a "basic logging" and a "tracking service" API. The "basic logging" API deals with logging metrics, parameters, and artifacts to the currently-active active run, and is accessible in ``mlflow`` (e.g., ``mlflow.log_param``). The tracking service API allow managing experiments and runs (especially historical runs) and is available in ``mlflow.tracking``. The tracking service API will look analogous to the upcoming R and Java Tracking Service SDKs. Please be aware of the following breaking changes:
+
+  - ``mlflow.tracking`` no longer exposes the fluent API, only ``mlflow``. So, code that was written like ``from mlflow.tracking import log_param`` will have to be ``from mlflow import log_param`` (note that almost all examples were already doing this).
+  - Access to the service API goes through the ``mlflow.tracking.get_service()`` function, which relies on the same tracking server set by either the environment variable ``MLFLOW_TRACKING_URI`` or by code with ``mlflow.tracking.set_tracking_uri()``. So code that used to look like ``mlflow.tracking.get_run()`` will now have to do ``mlflow.tracking.get_service().get_run()``. This does not apply to the fluent API.
+  - ``mlflow.ActiveRun`` has been converted into a lightweight wrapper around ``mlflow.entities.Run`` to enable the Python ``with`` syntax. This means that there are no longer any special methods on the object` returned when calling ``mlflow.start_run()``. These can be converted to the service API. For example, ``mlflow.start_run().set_terminated()`` would now be called as:
+
+    .. code:: python
+
+        import mlflow
+        run = mlflow.start_run()
+        mlflow.tracking.get_service().set_terminated(run.info.run_uuid)
+
+    (Note that in a future version, we will likely rename or alias ``run_uuid``.)
```

Comment:
Not sure that we need the example or the note here. I don't think people were calling methods on the returned objects super often.